src/cljs/harja/tiedot/hallinta/integraatioloki.cljs

```diff
@@ -45,6 +45,8 @@
 (defonce valittu-aikavali (atom nil))
 (defonce hakuehdot (atom {:tapahtumien-tila :kaikki}))
 (defonce nayta-uusimmat-tilassa? (atom true))
+;; tapahtuma-id atomia käytetään vain, kun tullaan Släckistä integraatiologiin
```

Comment:
Kommenttia voisi hieman laventaa, oisko "kun seurataan ulkoista integraatiolokiin linkkaavaa urlia - näitä lokitetaan ja linkin voi avata suoraan slackista"